site/templates/subsciptions.yaml

```diff
@@ -0,0 +1,40 @@
+{{- range .Values.site.subscriptions }}
+{{- $subs := . }}
+{{- $installPlanValue := .installPlanApproval }}
+
+{{- if $subs.namespaces }}
+{{- range .namespaces }}
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: {{ $subs.name }}
+  namespace: {{ default "openshift-operators" . }}
+spec:
```

Comment:
I'm a little twitchy about some of these defaults (there seems to be a lot of divergence around channel names in particular, and likewise source and sourceNamespace might be a bit of a surprise...but if someone accepts the defaults and they're wrong, they won't get anything installed, and it's not like we can validate it for them.

I'm OK with this (LGTM)

Reply:
@mhjacks Indeed. The default values for namespace are not ideal.  Not all operators will be deployed/installed under the openshift-operators namespace.  That said, we need to document this and make sure that we explicitly say that if a namespace is not specified that the operator will attempt to install under the openshift-operator namespace.